stable/prometheus/templates/alertmanager-clusterrolebinding.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app: {{ template "prometheus.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: "{{ .Values.alertmanager.name }}"
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "prometheus.alertmanager.fullname" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "prometheus.alertmanager.fullname" . }}
+    namespace: {{ .Release.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
This still strikes me as dangerous. If an administrator installs this chart into a namespace in which other non-administrators can run pods, those other users can run a pod using this service account, and escalate their privileges up to "cluster-admin." I think you should at least document this hazard if you're not willing to come up with a safer, narrower _ClusterRole_ for Alertmanager here.